src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +10,13 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private static UserService service = new UserService();
+
     public boolean login(String email, String password) {
+        if (service.findByEmail(email) != null
+                && password.equals(service.findByEmail(email).getPassword())) {
+            return true;
+        }
```

Comment:
in this case, you call the findByEmail method twice - this can affect performance.